doc/api/perf_hooks.md

```diff
@@ -1276,6 +1276,36 @@ obs.observe({ entryTypes: ['function'], buffered: true });
 require('some-module');
 ```
 
+### Trace http by perf\_hooks
+
+The following example traces http client(OutgoingMessage) and
+http request(IncomingMessage):
```

Comment:
What does it trace? The performance? In any case HTTP should be uppercase I think:
```suggestion
The following example traces HTTP client (`OutgoingMessage`) and
HTTP request (`IncomingMessage`):
```